src/components/cluster_detail/cluster_apps.js

```diff
@@ -64,11 +59,21 @@ class ClusterApps extends React.Component {
       logoUrl: '/images/app_icons/chart_operator@2x.png',
       category: 'management',
     },
+    'metrics-server': {
+      name: 'metrics-server',
+      logoUrl: '/images/app_icons/metrics_server@2x.png',
+      category: 'management',
```

Comment:
IMO metrics-server is essential (e.g. for HPA) and we do not use it for our mgmt

sidenote, it seems we are not clear abotu essential vs mgmt, for me latter should be only about the things that are for our mgmt and not for the user, for me RBAC is essential, without it there's no security, we cannot remove that without influencing the customer, if we add or remove anything from mgmt that would not influence the customer directly.

Reply:
agreed and committed